xlml/utils/metric.py

```diff
@@ -567,5 +659,5 @@ def process_metrics(
 
   print("Test run rows:", test_run_rows)
 
-  if is_valid_entry():
+  if is_valid_entry() or task_metric_config.write_results_to_bigquery:
```

Comment:
What's an example metric that's not a valid entry but `write_results_to_bigquery=True`?

Reply:
Currently, writing metrics to BigQuery is only allowed for prod runs in the `cloud-ml-auto-solutions` project. Using `write_results_to_bigquery=True` will allow manual sweep runs in other projects like `tpu-prod-env-multipod` to write metrics to BigQuery.

Reply:
Can we move the logic into `is_valid_entry()`? Basically we whitelist the manual sweep runs.

Reply:
Done!